crates/rune/src/ast/item.rs

```diff
@@ -81,7 +88,14 @@ impl Item {
 
 impl Peek for Item {
     fn peek(t1: Option<ast::Token>, t2: Option<ast::Token>) -> bool {
```

Comment:
These peek impls should probably be demoted to not be implementations of `Peek`, and just associated method (e.g. remove this and just rely on `peek_as_stmt`). Because they won't currently match items with attributes which will mess with parsing in macros.